### ISSUE TITLE
DO NOT INSTALL - if you dont know what you are doing - might brick your computer

### ISSUE THREAD

**Issue by alikim-com** [DavidXanatos/DiskCryptor#69]
After installing (win10 home) asks to restart Windows. After restart the computer doesn't boot up anymore and gets stuck in autorepair mode and you won't be able to start Windows at all in any mode.
Highly malicious software.

**Comment by hardhub** [comment]
You need to fix your driver `hands.sys`. Be happy!

It works on MBR and GPT partitions for years (BIOS and UEFI beta as well).

**Comment by isihussain** [comment]
USB via crypt give error, and close program.

**Comment by DavidXanatos** [comment]
The tool works for me just fine, and it is clearly marked as beta, proceed at your own risk, 
I'm sure if you install it on a clean system with disabled secure boot it will work just fine.
Or set up your own secure boot manually.

Since you say only installation was enough to break your windows, first try disabling secure boot.

**Comment by alikim-com** [comment]
> I'm sure if you install it on a clean system with disabled secure boot it will work just fine.

For me it didn't, so don't be so sure

And don't add meaningless and cheesy "if you dont know what you are doing" piece into the title, what exactly I'm supposed to know?

If you care to look at the issues, you will see I'm not the only one whos computer was damaged by this software.

Your program is malicious, I'm reporting it,

**Comment by DavidXanatos** [comment]
The installer clearly states:
![grafik](https://user-images.githubusercontent.com/3890945/232194330-21f373e2-b229-40ef-a2f9-b1eed131bd92.png)

try loading the system hive from a recovery environment and remove references to DC in the filter drvier config

HKLM, SYSTEM\CurrentControlSet\Control\Class{71A27CDD-812A-11D0-BEC7-08002BE2092F}, LowerFilters, 0x00010008, dcrypt

HKLM, SYSTEM\CurrentControlSet\Control\Class{4D36E965-E325-11CE-BFC1-08002BE10318}, UpperFilters, 0x00010008, dcrypt

for many people it works just fine me included, so it is something particular to your setup that prevents it from working.

**Comment by DavidXanatos** [comment]
You can follow this simple guide to resolve the issue: https://github.com/DavidXanatos/DiskCryptor/issues/70

**Comment by DavidXanatos** [comment]
@alikim-com look the Installer clearly says its beta proceed at your own risk, and the download page tells you extra to have a backup on hand. 
Ignoring all warnings and then complaining that apparently the warnings were right, is not ok.
If you would have a backup you would just apply it and then post something along the lines 
"Beta 3 does not seam to be compatible with my system, [enter details], that's what I already tried to get it working [...], does anyone has any ideas what may be the issue and if it can be worked around or if i need to wait for a beta 4?"
That's a proper issue report.
Calling it Malicious however is completely out of place, Malicious implies intent which is here completely absent.
At the time when beta 3 was released it was working completely fine, MSFT changed enough around windows to make it no longer work for some configurations, for others it still works just fine.

Wait for beta 4 and test it safely with a backup on hand.
